### ISSUE TITLE
feature request - force reset connection to relays

### ISSUE THREAD

**Issue by GretaFrank** [hzrd149/nostrudel#53]
as you said the connection gets dropped from time to time and notifications don't show "until something resets it" ...

well, why not give us a button that resets it ?  if i am talking to somebody and i feel like they have probably replied but i'm not getting the reply it would be nice to have that button.

i have already seen it a few times when i am waiting for reply, it doesn't come, then i go to timeline and it's in my timeline - but it wasn't in the notifications ...

basically a manual hard reset of connection to relays option ...

**Comment by hzrd149** [comment]
In a way the browsers "reload page" button is a force reset. although I intend to fix this issue when I rebuilt the notifications view
